dojo/templates/dojo/view_user.html

```diff
@@ -106,7 +106,7 @@ <h4 class="pull-left">{% trans "Contact Information" %}</h4>
                 <div class="clearfix">
                     <h4 class="pull-left">{% trans "Product Types this User can access" %}</h4>
                     &nbsp;
-                    <a href="https://documentation.defectdojo.com/usage/permissions/" target="_blank">
+                    <a href="https://docs.defectdojo.com/en/customize_dojo/user_management/about_perms_and_roles/#productproduct-type-membership--roles" target="_blank">
```

Comment:
Same here?

Reply:
Same for here, nothing can be combined. There is one documentation link that explains Product and Product Type membership for Users, but two separate places in the UI to add Product and/or Product Type membership to a user (which cannot be combined).